app/helpers/snowcrash_helper.rb

```diff
@@ -13,18 +13,18 @@ def css_class_for_sub_nodes(node)
   end
 
   def flash_messages
-    flash.collect do |name, msg|
-      flash_css = 'alert'
+    # In general controllers use :error, but :alert is used with redirect_to
+    #   http://guides.rubyonrails.org/action_controller_overview.html#the-flash
+    alert_types = {
+      'alert'   => ' alert-error',
+      'error'   => ' alert-error',
+      'info'    => ' alert-info',
+      'notice'  => ' alert-success',
+      'warning' => ''
+    }
 
-      # In general controllers use :error, but :alert is used with redirect_to
-      #   http://guides.rubyonrails.org/action_controller_overview.html#the-flash
-      flash_css << {
-        'alert'   => ' alert-error',
-        'error'   => ' alert-error',
-        'info'    => ' alert-info',
-        'notice'  => ' alert-success',
-        'warning' => ''
-      }[name]
+    flash.select { |key, _| alert_types.keys.include?(key) }.collect do |name, msg|
```

Comment:
Whilst I always appreciate a good one-liner, I'm curious if this approach wins us anything over the original?